### DISCUSSION TITLE
When I use a ScrolledFrame, how can I dynamically adjust the value of scrollheight according to the length of the content?

### DISCUSSION THREAD

**Discussion by wuliweigame** [israel-dryer/ttkbootstrap#597] [Q&A] [answered]
cost_frame = ScrolledFrame(frame1,height=600,width=430,autohide=True,scrollheight=800)

**Comment by israel-dryer** [comment, marked as answer]
`scrollheight` is an optional **override**, and its default (`None`) already gives you the "size to content" behavior you're after — so the fix is to simply not set it:

```python
# Let the content height track the contents (this is the default)
cost_frame = ScrolledFrame(frame1, height=600, width=430, autohide=True)
```

How it works: the content frame is placed with `place(..., height=scrollheight)`. When `scrollheight=None`, Tk uses the frame's *requested* height, which grows automatically as you add children, and the scrollbar thumb is computed from that live height. By passing `scrollheight=800` you pin a static height and opt out of the dynamic sizing.

If you ever need to set/clamp the height at runtime (e.g. after reloading data), the content frame's `place` methods are exposed with a `content_` prefix:

```python
cost_frame.content_place_configure(height=new_height)  # set a specific height
cost_frame.content_place_configure(height="")          # back to auto-size-to-content
```

That's the whole thing — dropping `scrollheight` is the fix.
